openedx/core/djangoapps/call_stack_manager/tests.py

```diff
@@ -1,16 +1,17 @@
 """
 Test cases for Call Stack Manager
 """
+import collections
 from mock import patch
 from django.db import models
 from django.test import TestCase
 
-from openedx.core.djangoapps.call_stack_manager import donottrack, CallStackManager, CallStackMixin
+from openedx.core.djangoapps.call_stack_manager import donottrack, CallStackManager, CallStackMixin, trackit
+from openedx.core.djangoapps.call_stack_manager import core
 
 
 class ModelMixinCallStckMngr(CallStackMixin, models.Model):
-    """
-    Test Model class which uses both CallStackManager, and CallStackMixin
+    """ Test Model class which uses both CallStackManager, and CallStackMixin
```

Comment:
As a summary line.
I did not write extended description as it does not have any.

Following this standard -
```
def func(arg1, arg2):
    """Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value

    """
```